README.md

```diff
@@ -0,0 +1,30 @@
+phpspec
+=======
+
+The main website with documentation is at `http://www.phpspec.net <http://www.phpspec.net>`_.
+
+[![Master Travis Build Status](https://travis-ci.org/phpspec/phpspec.svg?branch=master)](https://travis-ci.org/phpspec/phpspec)
+[![Master Scrutinizer Quality Score](https://img.shields.io/scrutinizer/g/phpspec/phpspec.svg)](https://scrutinizer-ci.com/g/phpspec/phpspec/build-status/master)
+[![AppVeyor build status](https://img.shields.io/appveyor/ci/ciaranmcnulty/phpspec/master.svg)](https://ci.appveyor.com/project/ciaranmcnulty/phpspec/branch/master)
+[![Total downloads](https://poser.pugx.org/phpspec/phpspec/downloads.png)](https://poser.pugx.org/phpspec/phpspec/downloads)
+[![Latest stable version](https://poser.pugx.org/phpspec/phpspec/v/stable.png)](https://poser.pugx.org/phpspec/phpspec/v/stable)
+[![Latest unstable version](https://poser.pugx.org/phpspec/phpspec/v/unstable.png)](https://poser.pugx.org/phpspec/phpspec/v/unstable)
+[![Software license](https://poser.pugx.org/phpspec/phpspec/license.png)](https://poser.pugx.org/phpspec/phpspec/license)
+
+
+Installing Dependencies
```

Comment:
Similar to the main heading, but you should use two hashes here, so it should be: `## Installing Dependencies` without any underlining.